docs/Reference/Warnings.md

```diff
@@ -0,0 +1,36 @@
+
+<h1 align="center">Fastify</h1>
+
+## Warnings
+
+### Warnings In Fastify
+Warnings are enabled by default, and you can disable it by:
+
+- setting the `NODE_NO_WARNINGS` environment variable to `1`
+- passing the `--no-warnings` flag to the node process
+- setting 'no-warnings' in the `NODE_OPTIONS` environment variable
+
+However, disabling warnings is not recommended as it may cause
+potential problems when upgrading Fastify versions.
+Only experienced users should consider disabling warnings.
+
+### Fastify Warning Codes
+
+| Code | Description | How to solve |
+| ---- | ----------- | ------------ |
+| **FSTWRN001** | The specified schema for a route is missing. This may indicate the schema is not well specified. | Check the schema for the route. |
+| **FSTDEP005** | You are accessing the deprecated `request.connection` property. | Use `request.socket`. |
```

Comment:
I think warnings and deprecations should be listed in separate tables. Deprecations are slated to be removed in future versions. Warnings are permanent(ish).

It would also be good to link to the PR/Issue that introduced the warning/deprecation as the discussion can provide insight into the change and solution.